Automattic/simplenote-android
# Add final details to Import Data Feature

Fixes #1417

### Fix
This PR is a follow-up on #1333 to add final details to the feature of importing notes in simplenote-android. 

| Importing one note (Marknown)   | Importing Several Notes (JSON) |
| ----------- | ----------- |
| ![20210719_141640](https://user-images.githubusercontent.com/195721/126217273-a92f4509-7fcd-4193-9e15-98d763cc32ce.gif) |  ![20210719_141743](https://user-images.githubusercontent.com/195721/126217301-0c6f691c-5aee-4ae4-8ed0-89bc13c43750.gif) |

### Test

1. Go to the top-left corner button and tab to open the menu
2. Tap on `Settings`
3. Scroll down and tab on `Import data`
4. Select a file with extensions `.txt`, `.md` or `.json`. The `.json` should be exported from a Simplenote app.
5. Go to All Notes and verify that the note or notes were imported

### Review

Only one developer and one designer are required to review these changes, but anyone can perform the review.

### Release

`RELEASE-NOTES.txt` was updated in ba7a22929cfc9c857d06427bb6a97b8a3feef2d3 with:
> Added support to import notes
